daras_ai_v2/settings.py

```diff
@@ -44,6 +44,9 @@
 CSRF_COOKIE_SECURE = config("CSRF_COOKIE_SECURE", cast=bool, default=False)
 SESSION_COOKIE_SECURE = config("SESSION_COOKIE_SECURE", cast=bool, default=not DEBUG)
 
+# CSP settings
+CSP_DEFAULT_SRC = ("*",)
```

Comment:
This is deprecated? https://django-csp.readthedocs.io/en/latest/migration-guide.html

Reply:
I also think we should do our own headers middleware for this so we can share them between fastapi and django but that depends on what exactly they are asking for protection against. My take was that we right now only need "Content-Security-Policy": "frame-ancestors 'none'" to protect against clickjacking

Reply:
i don't have clarity on what they want. `default-src: "*"` does not protect against anything, it just adds the header and says explicitly that "any src is allowed, we know what we're doing and that's what we want".

the `django-csp` library defaults `default-src` to `"'self'"` - which would block external assets from being loaded. so i changed that to the most minimal change we would need without messing up anything.

we could also do a middleware change, i do like that more. this was just faster

Reply:
ok django already does XFrameOptionsMiddleware so lets do this for now and let it be